apps/web/tsconfig.json

```diff
@@ -6,9 +6,11 @@
     "moduleResolution": "Bundler",
     "jsx": "preserve",
     "typeRoots": ["./node_modules/@types"],
+    "baseUrl": ".",
```

Comment:
is this needed?

Reply:
Not sure. Did many things on friday. I'll clean up in the next PR when it's working

Reply:
ok don't forget to remove it please